main.go

```diff
@@ -88,7 +75,105 @@ func main() {
 			log.Error(err)
 		}
 	})
+	if *allClouds {
+		http.HandleFunc("/probe", probeHandler)
+		http.Handle("/metrics", promhttp.Handler())
+	} else {
+		http.HandleFunc(*metrics, metricHandler(services))
```

Comment:
Please add a log entry that specifies that is running in legacy mode

Reply:
Done